### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by SergioGasquez**
I haven't been able to test it in my fork, but basically they are copied from https://github.com/esp-rs/espup/tree/main/.github/ISSUE_TEMPLATE where it works fine

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Steps to reproduce the behavior:
+
+1. ...
+2. ...
+3. ...
+4. ...
+
+<!-- Make sure you are able to reproduce the bug in the main branch, too. -->
+<!-- Make sure the issue is not already reported. -->
```
Comment (MabezDev) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:26]: Maybe we can make this a tick box option, like "I have searched the existing issues (closed & open) to make sure the issue hasn't already been reported?"

Comment (SergioGasquez) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: Good idea! I've updated the comments to checkboxes

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Screenshots
+
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+## Environment
+
+<!-- Please fill the following information. -->
+
+- OS: [e.g. Ubuntu 20.04]
+- esfpaslh/cargo-espflash version: [e.g. 0.1.0]
```
Comment (jessebraham) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:41]:
```suggestion
- espflash/cargo-espflash version: [e.g. 0.1.0]
```

**Review by jessebraham — CHANGES_REQUESTED** [review]
Just one small typo but otherwise LGTM, thanks for adding these!

**Review by jessebraham — APPROVED** [review]

**Pull request merged**